.gitignore

```diff
@@ -6,6 +6,7 @@
 dist/
 *.egg-info
 *.egg
+*.eggs/
```

Comment:
You may also need to add a `.cache/` here as well, since pytest creates a local cache.